action/pipeline/pipeline.go

```diff
@@ -16,6 +16,7 @@ type Config struct {
 	Target           string
 	Org              string
 	Repo             string
+	Steps            []string
```

Comment:
```suggestion
	SkipSteps            []string
```
Thoughts on making this field more descriptive?